README.md

```diff
@@ -175,6 +192,23 @@ needs to be done that I haven't documented in the issues yet, however. For more
 ideas or help with running or hacking on Miri, you can contact me (`scott`) on
 Mozilla IRC in any of the Rust IRC channels (`#rust`, `#rust-offtopic`, etc).
 
+## History
+
+This project began as part of an undergraduate research course in 2015 by
```

Comment:
cc @solson

Reply:
Should we wait for @solson to appear, or incorporate their feedback when it comes?